locale/en/LC_MESSAGES/django.po

```diff
@@ -7232,3 +7232,23 @@ msgstr "Download client"
 #:
 msgid "Powered by Open365"
 msgstr "Powered by Open365"
+
+#: seahub/templates/view_shared_dir.html:13
+#: seahub/templates/shared_file_view.html:17
+msgid "File shared using <strong>Open365</strong> personal cloud. "
+msgstr "File shared using <strong>Open365</strong> personal cloud. "
```

Comment:
`en` translations should leave the `msgstr` blank, as the translation is the same as the key.